test/run_as_user/run_as_user_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build linux && integration
+// +build linux,integration
+
+package run_as_user
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/aws/amazon-cloudwatch-agent-test/test"
+)
+
+const (
+	// Let the agent run for 15 seconds. This will give agent enough time to change user
+	agentRuntime     = 15 * time.Second
+	configOutputPath = "/opt/aws/amazon-cloudwatch-agent/bin/config.json"
+	pidFile          = "/opt/aws/amazon-cloudwatch-agent/var/amazon-cloudwatch-agent.pid"
+	root             = "root"
+	cwagent          = "cwagent"
+)
+
+type input struct {
+	user      string
+	dataInput string
+}
+
+func TestBundle(t *testing.T) {
+
+	parameters := []input{
+		{dataInput: "resources/default.json", user: root},
```

Comment:
Are there any clashing issues we need to aware about? Same as `ca_bundle`. If yes, are there any ways we could prevent it or improve it further.

Reply:
No since we do run tests one by one.